web-registry/src/generated/json-ld/cfc-project-metadata.ts

```diff
@@ -0,0 +1,24 @@
+import { URL } from 'web-components/lib/types/rdf';
+
+import { ProjectMetadataLD } from './project-metadata';
+
+// type generated from https://github.com/regen-network/regen-registry-standards/blob/main/jsonld/projects/C02-project.jsonld
+
+export interface CFCProjectMetadataLD extends ProjectMetadataLD {
+  'regen:cfcProjectId'?: string;
+  'regen:cfcProjectPage'?: string;
+  'regen:projectDesignDocument'?: string;
```

Comment:
@wgwz - Talked to @S4mmyb about this. Do you think this should be a NameUrl type, so that we can display a name instead of just always "Project Design Document" in the UI link?